pkg/cmd/pulumi/convert.go

```diff
@@ -95,27 +95,48 @@ func newConvertCmd() *cobra.Command {
 	return cmd
 }
 
-// pclGenerateProject writes out a pcl.Program directly as .pp files
-func pclGenerateProject(directory string, project workspace.Project, p *pcl.Program) error {
-	if directory != "." {
-		err := os.MkdirAll(directory, 0o755)
+func writeOutPclFiles(directory *pcl.ProgramDirectory) error {
+	if _, err := os.Stat(directory.Path); os.IsNotExist(err) {
+		// directory does not exist, create it
```

Comment:
You can just always call mkdiral, no need to stat first.

Reply:
Updated in last commit, now using `MkdirAll` ✅